_includes/speakers.html

```diff
@@ -1 +1 @@
-{% assign slugs=include.speakers|split:" " %}{% for slug in slugs %}{% assign speaker = site.data.programme.speakers[slug] %}{{ speaker.name }}{% unless forloop.last %}, {% endunless %}{% endfor %}
+{%- assign slugs=include.speakers|split:" " -%}{%- for slug in slugs -%}{%- assign speaker = site.data.programme.speakers[slug] -%}{{ speaker.name }}{%- unless forloop.last -%}, {%- endunless -%}{%- endfor -%}
```

Comment:
The reason I had to do this is that otherwise an empty speakers list would come back as `"\n"`, which would not be caught by the `{{ unless speakers == '' }}` and we'd end up with an event description of `"Featuring ."` (instead of skipping the "Featuring" block altogether).